ruby/README.md

```diff
@@ -49,12 +33,12 @@ Credentials Reference Guide](https://docs.aws.amazon.com/credref/latest/refdocs/
 * bundler 2.3.8 or later (to use bundler to install gems)
 
 ## Tests
+**Note**: Running the tests might result in charges to your AWS account.
 
-All tests use RSpec, and you can find them in the `spec` folder for each example.
-When an example has additional requirements to run tests, you can find them in the
-README for that service or cross-service example.
+All tests use RSpec, and you can find them in the `spec` folder for each example. Please check the README for each service or cross-service example for any additional requirements to run tests.
```

Comment:
The template does say "Please check..." but I think AWS style is that we don't say "please" in tech docs. I don't feel strongly about this, so it's up to you.

Reply:
Good question. I'll leave it be until we update the template. Added it to retro items.